app/services/scoreboard_all.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# scoreboard for all schools
+class ScoreboardAll
+  include Scorable
+
+  SLUG = 'all'
+
+  attr_reader :academic_year_calendar
+
+  def initialize
+    @academic_year_calendar = Calendar.find_by(title: 'England and Wales', calendar_type: 'national')
```

Comment:
This isn't ideal but I guess the alternative would be to have an attribute on the Calendar model to identify the default calendar to use for the national scoreboard, which doesn't seem much better.

Currently the national calendars can't be deleted or renamed, so this seems fairly safe.